js/adapt-youtube.js

```diff
@@ -44,19 +42,19 @@ define([
 
       $iframe.width(widgetWidth);
 
-      var aspectRatio = (this.model.get('_media')._aspectRatio ? parseFloat(this.model.get('_media')._aspectRatio) : 1.778);//default to 16:9 if not specified
-      if (!isNaN(aspectRatio)) {
-        $iframe.height(widgetWidth / aspectRatio);
-      }
+      var aspectRatio = (this.model.get('_media')._aspectRatio ? parseFloat(this.model.get('_media')._aspectRatio) : 1.778);// default to 16:9 if not specified
+      if (isNaN(aspectRatio)) return;
+
+      $iframe.height(widgetWidth / aspectRatio);
     },
 
     postRender: function() {
-      //FOR HTML/HBS Paramenters: https://developers.google.com/youtube/player_parameters
+      // FOR HTML/HBS Paramenters: https://developers.google.com/youtube/player_parameters
```

Comment:
Typo: `Paramenters`.

Reply:
TBH not sure that comment is particularly useful in that location... might remove